### PR TITLE
🧬 Sell token modal

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3836

### CODE REVIEW

File: packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+      {
+        title: 'You will get',
+        content: (
+          <NumberFormat
+            value={tokens || 0}
+            as="p"
+            variant="t200"
+            withDenomination="before"
+            withToken
+            customTicker="$JBC"
```
Comment (attemka) [inline packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx]: you're selling $JBC here, so you should receive `${atlasConfig.joystream.tokenTicker}`?

Comment (attemka) [reply packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx]: same thing with align as in token buy modal

**Review by attemka — CHANGES_REQUESTED** [review]

File: packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx (hunk 1 above)
Comment (thesan) [inline packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx]:
```suggestion
            customTicker={`$${title}`}
```

File: packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+            content={pricePerUnit}
+            icon={<JoyTokenIcon size={smMatch ? 24 : 16} variant="silver" />}
+            withDenomination
+          />
+          <DetailsContent
+            avoidIconStyling
+            tileSize={smMatch ? 'big' : 'bigSmall'}
+            caption={`YOUR $${title} BALANCE`}
+            content={userBalance}
+            icon={<JoyTokenIcon size={smMatch ? 24 : 16} variant="silver" />}
```
Comment (thesan) [inline packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx:86]:
```suggestion
```

But also if I understand correctly `userBalance` is in $JBC but the $USD conversion is based on the current $JOY/$USD rate so the $USD value will be wrong.

Comment (WRadoslaw) [reply packages/atlas/src/components/_crt/SellTokenModal/SellTokenModal.tsx:86]: I made it a separate issue #4761

**Review by attemka — APPROVED** [review]

**Pull request merged**